### ISSUE TITLE
`yalc add <package_name>` should install dependencies of package and install them.

### ISSUE THREAD

**Issue by luk707** [whitecolor/yalc#81]
I've only recently switched to using `yalc` over `yarn link` and it's been super useful for fixing issues such as duplicate deps etc. Through my use over the last couple of days I've noticed that every time I link a package I'm working on to another project it does not install it's deps. Say I have a package with some deps of it's own and run `yalc add <package_name>`, I have to immediately run `yarn` to get it's deps to appear in my project. I feel like the `yalc add` command was purposefully designed to mimic the experience of `yarn add` thus I feel the correct behaviour after running `yalc add <package_name>` would be for it to run `yarn` or `npm` afterward.

**Comment by whitecolor** [comment]
well yes, it could. 

Yalc actually designed for its purpose, its not supposed to replace tool. Need to think if its worth to run `yarn` by default. 

Btw, If you use `--yarn` option it should run. it is not much better then `yalc add .. && yarn` -)

**Comment by robbiehudson** [comment]
This was confusing to me too. I "expected" the dependencies to be installed automatically too and had to dig around to find out this was the problem.

**Comment by whitecolor** [comment]
@robbiehudson should read the docs more carefully

**Comment by robbiehudson** [comment]
@whitecolor absolutely! But it would also be clearer if the warning wasn't in a separate section. Perhaps an example flow in the Readme would be useful? Happy to create a PR if you think that'd be useful.

**Comment by jiaolongshenme** [comment]
yarn.lock file in my project  . when i use `yalc add XXX ` and then `yarn`, the deps of XXX does not install
